modules/tensor_mechanics/src/materials/Compute2DIncrementalStrain.C

```diff
@@ -0,0 +1,52 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
```

Comment:
It's unclear from the code standards if the header should have a space after or not; please let me know how you prefer the spacing to be handled and I will change the files accordingly.